contracts/token/psp721/src/traits.rs

```diff
@@ -291,19 +291,27 @@ pub trait IPSP721: PSP721Storage {
 
     fn _mint(&mut self, id: Id) {
         let to = Self::env().caller();
+        self._mint_to(to, id);
+    }
+
+    fn _mint_to(&mut self, to: AccountId, id: Id) {
         self._add_to(to, id.clone());
         self._emit_transfer_event(ZERO_ADDRESS.into(), to, id);
     }
 
+    fn _burn_from(&mut self, from: AccountId, id: Id) {
+        self._remove_from(from, id.clone());
```

Comment:
would be great if it'd be possible to get rid of `.clone()` here; maybe accept the id param by reference, and if cloning is needed, do it on a higher level to be more explicit?